### ISSUE TITLE
Feature: release stable releases more often according to semver

### ISSUE THREAD

**Issue by mcmxcdev** [ferdium/ferdium-app#542]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/ferdium/ferdium-app/blob/develop/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-app/blob/develop/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/ferdium/ferdium-app/issues) for a feature request that matches the one I want to file, without success.

### Problem Description

I regularly recommend Ferdium to people I know and send them to download it (like I want to right now)

Unfortunately, the website only contains the stable v6.0.0 (as of now) release which is 15 days old at this point (For Ferdium which has such a high nightly release cadence, that makes quite the difference).

Would it be possible to stick at least a bit more to semver and release features as e.g. 6.1.0 or important bug fixes as 6.0.1 regularly?

### Proposed Solution

Release more often to the public!

### Alternatives Considered

Keep the status quo which also leads to people reporting bugs that might have been fixed already in nightly versions.

**Comment by remss** [comment]
I also wish to have more frequent release, just to get a recent Electron/Chrome version bump. My company has strict policy to allow only the latest stable browser version to access the services. So currently I cannot use Ferdium because the service sees an old version of Chrome ☹️

**Comment by mcmxcdev** [comment]
Oh, that's too bad, sorry that you cannot use it currently!

I am following along with the Electron v28 release anyway, but wanted to wait for some patch version release first. Once the new major release stabilized, I would upgrade it and we would draft a new release.

**Comment by remss** [comment]
Thank you! 
I have the same issue with latest 6.6.1-nightly.14 build but building myself from sources the 6.6.1-nightly.17 solves my problem. Publishing nightly build more often could be a solution?

**Comment by mcmxcdev** [comment]
Sure thing, but that would need more maintainers wiling to spend the time and do it. We all do it in our spare time, so releases will probably continue to happen spontaneously and not follow a consistent pattern.

**Comment by remss** [comment]
I agree that it should not take your time but nightly builds is a an automatic process right? So what it the issue to run it more frequently? I believe other users will not have the time/skill to build from sources.

**Comment by mcmxcdev** [comment]
Nightly builds are automatically generating drafts, but then they need a changelog of changes documented from both app and recipes as well as making sure they work before getting released.
